network/edge-services/how-to/configure-custom-domain.mdx

```diff
@@ -0,0 +1,123 @@
+      - Optionally, add tags by typing each tag and then pressing enter
+      - Copy and paste the full PEM-formatted certificate chain into the box.
+      Your certificate will be automatically stored in Secret Manager and [billed accordingly](https://www.scaleway.com/en/pricing/security-and-account/).
+
+    <Message type="tip">
+    For help with SSL/TLS certificates for Edge Services, and/or dealing with any errors you encounter importing a certificate into Secret Manager, see our [dedicated documentation](/network/edge-services/reference-content/ssl-tls-certificate/).
+    </Message>
+
+6. Click **Customize domain** to finish.
+
+Your customized domain is set up, and you are returned to the Edge Services dashboard. The customized domain displays in the Endpoint panel. When you access your Object Storage or Load Balancer origin through this domain, its content will be served via Edge Services.
+
+<Message type="note">
+If you chose to generate a managed Let's Encrypt certificate, allow a few minutes for the certificate to finish creating. When the process is complete and the certificate is ready, you will see a green status light for **SSL/TLS certificate** on your endpoint dashboard.
+
+<Lightbox src="scaleway-edge-services-certificate-complete.webp" alt="" /> TODO CHECK
+
+</Message>
+
+## How to edit your customized domain or its certificate
+
+After customizing your domain, you can edit it (or its certificate) at any time as follows:
+
+1. In the Scaleway console, navigate to the Edge Services dashboard for the Object Storage bucket or Load Balancer origin whose domain you want to customize:
+
+        TODO INSERT SCREENSHOT
+
+2. In the **Endpoint** panel, click **Edit**. The **Edit Domain** screen displays.
+
+3. Edit the subdomain as desired - do not forget to also set up a new CNAME record, if necessary.
+
+4. Edit your certificate options as required - choose to generate a managed Let's Encrypt certificate, managed by Scaleway including automatic renewals, or select a different certificate from Secret Manager, or manually import a new certificate for your custom domain.
+
+5. Click **Edit domain** to finish.
+
+## How to reset your customized domain
+
+Even though the original Edge Services endpoint (e.g. `https://pipeline-id-or-bucket-name.svc.edge.scw.cloud`) will continue to work after you add a customized domain, you can choose to remove your customized domain completely and go back to the original Edge Services endpoint only. This is done via the reset function:
+
+1. In the Scaleway console, navigate to the Edge Services dashboard for the Object Storage bucket or Load Balancer origin whose domain you want to reset:
+
+        TODO INSERT SCREENSHOT
+
+2. In the **Endpoint** panel, click **Reset**.
+
+    A screen displays warning you that this will reset the pipeline's domain back to the default Edge Services endpoint. Edge Services will consider your customized subdomain as unknown. You should also remember to:
```

Comment:
```suggestion
    A screen displays, warning you that this will reset the pipeline's domain back to the default Edge Services endpoint. Edge Services will consider your customized subdomain as unknown. You should also remember to:
```